.github/workflows/release-docker.yml

```diff
@@ -15,8 +14,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python_version: ["3.7", "3.8", "3.9"]
-        pytorch_version: ["1.9", "1.10"]
+        include:
+          # We only release one docker image per PyTorch version.
+          - {python_version: "3.9", pytorch_version: "1.9", cuda_version: "11.1.1"}
+          - {python_version: "3.9", pytorch_version: "1.10", cuda_version: "11.3.1"}
+          - {python_version: "3.9", pytorch_version: "1.11", cuda_version: "11.3.1"}
+          - {python_version: "3.9", pytorch_version: "1.12", cuda_version: "11.3.1"}
```

Comment:
@carmocca @Borda I reduced the number of PL release images to build/push in our CI/CD, but do you think these combinations are reasonable?

Before making this change, I updated the matrix as follows:
```diff
        python_version: ["3.7", "3.8", "3.9"]
-       pytorch_version: ["1.9", "1.10"]
+       pytorch_version: ["1.9", "1.10", "1.11", "1.12"]
```
but I felt like these combinations were too many to maintain, so I pruned most of them in cd4a107815bba9cb3e4b0f03a161e06cb27ce668.

Reply:
I agree with the pruning. My suggestion is broad coverage for PyTorch versions, and thin coverage for Python versions. Very rarely does something fail because of a Python version issue.

So 1 job per Python version is fine with me.